src/config.js

```diff
@@ -139,7 +139,13 @@ global.config = {
       1800: [8, 15], // RCL 5
       2300: [11, 21], // RCL 6
     },
+    transferToCarry: false,
+    // limit carry spawn rate when RCL < 4
     minSpawnRate: 50,
+    // when ticksToLive > recycleThreshold, reuse carry
+    recycleThreshold: 200,
+    // keep at least one carry for a target
+    ensureOne: true,
```

Comment:
I couldn't think up a better name...

Reply:
I think it is fine, maybe `ensureOnePerTarget` ... :-)